core/storage/collection/gae_models.py

```diff
@@ -479,13 +505,6 @@ class CollectionCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
     collection_id = (
         datastore_services.StringProperty(indexed=True, required=True))
 
-    @staticmethod
-    def get_deletion_policy():
```

Comment:
Shouldn't CollectionCommitLogEntryModel be PSEUDONYMIZE_IF_PUBLIC_DELETE_IF_PRIVATE? Ditto for explorations.

Reply:
Yep, makes sense, done.